lib/vtls/schannel_verify.c

```diff
@@ -77,21 +77,150 @@ static int is_cr_or_lf(char c)
   return c == '\r' || c == '\n';
 }
 
-static CURLcode add_certs_to_store(HCERTSTORE trust_store,
-                                   const char *ca_file,
-                                   struct Curl_easy *data)
+static const char *c_memstr(const char *str, const char *str_limit,
+                            const char *sub, size_t size_sub)
```

Comment:
I added a comment. You can fix my poor english if needed. It is a clone of memmem not avaiable in VS2019

Reply:
I would rather you made it use the *same* arguments as `memmem` uses, since then we can use the provided memmem on the platforms that have it, and only use our custom function on the platforms that don't have it!

Reply:
I just commited c_memmem which is now a clone of memmem